presto-docs/src/main/sphinx/release/release-318.rst

```diff
@@ -0,0 +1,58 @@
+===========
+Release 317
+===========
+
+General Changes
+---------------
+
+* Fix ``DISTINCT FROM`` for ``UUID`` type. (:issue:`1180`)
+* Improve performance for queries over ``information_schema`` that have a limit. (:issue:`999`)
+* Improve query performance when ``optimize_hash_generation`` is enabled. (:issue:`1071`)
+* Add support for ``Asia/Qostanay`` time zone. (:issue:`1221`)
+* Add session properties for ``query_max_memory_per_node`` and
```

Comment:
The focus should be on what functionality was added, with a note on how to enabled it.

Reply:
@sopel39 ?

Reply:
```
* Add session properties to overwrite Query per-node memory limits: ``query_max_memory_per_node`` and
  ``query_max_total_memory_per_node``. These properties can be used to decrease limits for a query, but
  not to increase it. (:issue:`1212`)

Reply:
This is a duplicate of the above.